src/vtkweb/index.js

```diff
@@ -1,14 +1,59 @@
 import RemoteRenderer from 'ParaViewWeb/NativeUI/Canvas/RemoteRenderer';
 import { onSizeChange, startListening } from 'ParaViewWeb/Common/Misc/SizeHelper';
-
-export default function remoteRender(canvas, dataObject, graphicsMethod, template) {
+/**
+ * BEGIN simplified interface
+ * @param {string} dataSpec.file  data file where to read the variables from
+ * @param {string} dataSpec.variable variable or list of variables to plot (from dataSpec.file)
+ * @param {object} dataSpec.subset same as subset in dataSpec.variables.
+ *                 Subset is applied to all variables, so we assume
+ *                 all variables have the same grid.
+ * END simplified interface
+ * @param {object[]} dataSpec.variables       The variable from the file to display
+ *        A variable contains:
+ *          - name: the name of the attribute to plot
+ *          - file: the data file where to read the variable from
+ *          - subset: a list of dictionary elements, each element has
+ *              - name: index variable name
+ *              - range: list with lower and upper range for the index varia
+ */
+export default function remoteRender(canvas, dataSpec, template,
+                                     graphicsMethodType, graphicsMethodName) {
   return canvas.session.client()
     .then((client) => {
-      const renderer = new RemoteRenderer(client);
-      renderer.setContainer(canvas.el);
-      onSizeChange(() => {
-        renderer.resize();
-      });
-      startListening();
+      // simplified interface
+      if (dataSpec.file && dataSpec.variable) {
+        if (typeof dataSpec.variable === 'string') {
+          dataSpec.variables = [{
+            name: dataSpec.variable,
+            file: dataSpec.file,
+          }];
+        } else {
+          dataSpec.variables = [];
+          for (let i = 0; i < dataSpec.variable.length; ++i) {
```

Comment:
Note that you have variable and variables, so I think it the loop is fine. Not sure about the ++, my linter was happy with it. I guess we should make sure we force the linter version used so that we get consistent results. @jbeezley could you help?